services/tenant-ui/frontend/src/components/about/Traction.vue

```diff
@@ -3,10 +3,10 @@
     <div class="col-12 md:col-6">
       <strong>{{ $t('about.traction.info') }}</strong>
       <p class="my-0">
-        {{ $t('about.traction.tractionVersion', { version: '0.2.2' }) }}
+        {{ $t('about.traction.tractionVersion', { version: '0.1.1' }) }}
       </p>
       <p class="mt-0">
-        {{ $t('about.traction.uiVersion', { version: '0.2.2' }) }}
+        {{ $t('about.traction.uiVersion', { version: '0.1.1' }) }}
```

Comment:
This should not be hardcoded, but keeping for now. 
Will look at how to address with https://github.com/bcgov/traction/issues/648 among other things

Reply:
Wonder if this could be a compile-time substitution, potentially - IF we can pull it in from the `package.json`. Otherwise a dynamically-loaded config might be the way.

Reply:
Yeah the tenant UI one I'd like to pull from package.json.
Traction version I'm not actually sure where we define... could be the release version pulled in to dynamic config